airbyte-integrations/connectors/source-uk-police/source_uk_police/spec.yaml

```diff
@@ -0,0 +1,56 @@
+documentationUrl: https://docsurl.com
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: UK Police Spec
+  type: object
+  required:
+    - start_month
+    - lat
+    - lng
+  additionalProperties: true
+  properties:
+    start_month:
+      type: string
+      description: The month to start from
+      examples:
+        - 2019-01
+      pattern: "^[0-9]{4}-[0-9]{2}$"
+      order: 0
+    end_month:
+      type: string
+      description: The month to end at
+      examples:
+        - 2019-01
+      pattern: "^[0-9]{4}-[0-9]{2}$"
+      order: 1
```

Comment:
Rename this parameter, it is not a month. start_year_month

Reply:
Done